_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -515,8 +527,3 @@ def _capture_csrf_token(html):
     match = CSRF_REGEX.search(html)
     if match:
         return match.group(1)
-
-
-def _generate_random_respondent():
```

Comment:
Mark added this to make it more realistic, that different users are logging in, we don't want the same user all the time. He will also would of added 10 in the sample and enrolment for each which would now become redundant removing this

Reply:
I have restored this function.

Reply:
This pattern was already in place before I worked on these tests. However, it does make sense to randomly select registered users each time so this should stay in place.